### DISCUSSION TITLE
Do we have to show the Bloop stuff when you first run?

### DISCUSSION THREAD

**Discussion by ckipp01** [VirtusLab/scala-cli#802] [General]
If a newcomer is using `scala-cli` for the very first time and they want to run a project the first thing they'll see is:

```
❯ scala-cli run src/
Bloop daemon status: not running
Starting Bloop 1.4.19 at /Users/ckipp/Library/Caches/ScalaCli/bloop/daemon using JVM /Users/ckipp/Library/Caches/Coursier/arc/https/github.com/adoptium/temurin17-binaries/releases/download/jdk-17%252B35/OpenJDK17-jdk_x64_mac_hotspot_17_35.tar.gz/jdk-17+35/Contents/Home/bin/java
...
```

Just a thought, but I think that the fact that there is no Bloop daemon running will mean nothing to a new user. Then the info about it started up Bloop at a specific location is also an implementation detail. I can't find the message now, but at times it also shows a message about the jvm index being updating. I don't think any of this stuff should actually be a shown to a user when the first get started with the tool.

**Comment by lwronski** [comment]
Hi @ckipp01,

Right, those logs shouldn't be displayed for newcomers.  From the next version, scala-cli prints the following log:
```
Starting compilation server
```

instead of 

```
Bloop daemon status: not running
Starting Bloop 1.4.19 at /Users/foo/Library/Caches/ScalaCli/bloop/daemon using JVM /Users/foo/Library/Caches/Coursier/arc/https/github.com/adoptium/temurin17-binaries/releases/download/jdk-17%252B35/OpenJDK17-jdk_x64_mac_hotspot_17_35.tar.gz/jdk-17+35/Contents/Home/bin/java
```

It is already fixed in #700

**Reply by ckipp01** [reply]
Looking at the PR it seems that the following will still be displayed
```
Downloading JVM index
```
Which imo, the user doesn't need to know about.

**Reply by lwronski** [reply]
IMO too, user doesn't need to know about it.  So I turn off printing ```Downloading JVM index``` in #824 .

**Reply by ckipp01** [reply]
Awesome! Thanks for the quick feedback!

**Reply by lwronski** [reply]
It was already fixed in #824 , so I will close this discussion
